packages/flutter/test/material/theme_test.dart

```diff
@@ -254,13 +255,13 @@ void main() {
     );
     await tester.pump(const Duration(milliseconds: 100)); // Halfway through the theme transition
 
-    glyphText = tester.renderObject(find.byType(RichText));
+    glyphText = tester.renderObject(find.byType(RichText, skipOffstage: false));
```

Comment:
I can remove the `skipOffstage` parameters and convert them all to `find.all.blah(...)`. Would that be better?